src/Icons/Models/IconHttpResponse.cs

```diff
@@ -0,0 +1,83 @@
+﻿#nullable enable
+
+using System.Collections;
+using System.Net;
+using AngleSharp.Html.Parser;
+using Bit.Icons.Services;
+
+namespace Bit.Icons.Models;
+
+public class IconHttpResponse : IEnumerable<Icon>, IDisposable
```

Comment:
Should this still implement `IEnumerable<Icon>`. It doesn't look like `_icons` is ever set.